@here/harp-mapview/lib/poi/PoiManager.ts

```diff
@@ -44,6 +39,38 @@ interface ImageTextureDef {
     pixelRatio?: number;
 }
 
+function getPoiImageTexture(poiGeometry: PoiGeometry, index: number = 0): string | undefined {
```

Comment:
just another minor, maybe you can drop the *Poi* in the name as its inside the PoiManager, getImageTexture might be enough... same for the funtions below

Reply:
done